src/NSubstitute/NSubstitute.csproj

```diff
@@ -32,7 +32,6 @@
 
   <PropertyGroup>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <WarningsAsErrors />
```

Comment:
Doesn't look like formatting change. Is this intended?

Reply:
yes, this is cleanup unusable directive